store/modules/actions/authCore.js

```diff
@@ -1,7 +1,8 @@
 import { AuthCoreAuthClient } from 'authcore-js';
 import * as types from '@/store/mutation-types';
-import { AUTHCORE_API_HOST } from '@/constant';
+import { AUTHCORE_API_HOST, IS_TESTNET } from '@/constant';
 
+const authcoreURL = IS_TESTNET ? 'https://likecoin-integration-test.authcore.io' : 'https://authcore.like.co';
```

Comment:
Why do we need this now if we don't need it before?

Reply:
because I have "setupCosmosProvider" flow. If I delete "setupCosmosProvider", then this line could be deleted too! will work on deletion together